### ISSUE TITLE
Running main.py looks like doing training?

### ISSUE THREAD

**Issue by darthjaja6** [gsgen3d/gsgen#5]
![image](https://github.com/gsgen3d/gsgen/assets/139144277/b910cfff-074c-47f9-882f-d99f1fc3555b)
This shows up in log, I wonder if I did something wrong or just misunderstood. Isn't it supposed to be an inference?

**Comment by Phylliida** [comment]
I'm not a dev for this project but how this works is it starts with a rough shape given by Point-E, then uses stable diffusion (by default, u can change this in config) from different perspectives to guide Gaussian Splatting into the right shape. Their way of doing it takes a lot of iters to converge for a single 3D model (it takes me ~3 hours on a RTX 3090 to do 15000 steps). You can see the progress in outputs/<date>/<date>/checkpoints/<prompt>/<date>/<id>/eval, look in the image and video folders.

How frequently it shows you progress pictures/video is in base.yaml in eval, image_period and video_period

**Comment by darthjaja6** [comment]
@Phylliida Well explained, thank you!

**Comment by genitrix** [comment]
> TLDR: you're running inference for a single 3D model, it's just very slow
> 
> I'm not a dev for this project but how this works is it starts with a rough shape given by Point-E, then uses stable diffusion (by default, u can change this in base.yaml config guidance) from different perspectives to guide Gaussian Splatting into the right shape. Their way of doing it takes a lot of iters to converge for a single 3D model (it takes me ~3 hours on a RTX 3090 to do 15000 steps). You can see the progress in outputs/<date>/<date>/checkpoints/<prompt>/<date>/<id>/eval, look in the image and video folders.
> 
> How frequently it shows you progress pictures/video is in base.yaml in eval, image_period and video_period

Hi @darthjaja6, can you get the same result from the paper? I ran the training successfully but I got a result very different from my prompt.
